### ISSUE TITLE
Update nodes to Node-RED v1's new `input` event callback

### ISSUE THREAD

**Issue by cowchimp** [Nexmo/nexmo-nodered#46]
The syntax for listening to an incoming message and sending a message is changing in Node-RED v1.
The post about this from the official Node-RED blog explains why it's changing, what's the new syntax, and how to maintain backward compatibility:
https://nodered.org/blog/2019/09/20/node-done

Thanks.

**Comment by sammachin** [comment]
yes need to implement this for the 3.2 release of the package, will look into it more when I have some more capacity

**Comment by cowchimp** [comment]
Would you be interested in a PR that changes this based on the example from the blogpost?

**Comment by sammachin** [comment]
Yes please!


On 26 October 2019 at 09:29:18, cowchimp (notifications@github.com) wrote:

Would you be interested in a PR that changes this based on the example from the blogpost?
